File: segundos 4 lenguajes/c#/ProyectoCsharp/JugadorExperto.cs
namespace ProyectoJugadores
{
    public class JugadorExperto
    {
        public string Nombre { get; set; }
        public int Nivel { get; set; }

        public JugadorExperto()
        {
            Nombre = "Experto";
            Nivel = 20;
        }

        public JugadorExperto(string nombre, int nivel)
        {
            Nombre = nombre;
            Nivel = nivel;
        }

        public static JugadorExperto Copy(JugadorExperto j)
        {
            return new JugadorExperto(j.Nombre, j.Nivel);
        }
    }
}

File: segundos 4 lenguajes/js/JugadorExperto.js
export class JugadorExperto {
    constructor(nombre = "Experto", nivel = 20, copia = null) {
        if (copia) {
            this.nombre = copia.nombre;
            this.nivel = copia.nivel;
        } else {
            this.nombre = nombre;
            this.nivel = nivel;
        }
    }
}
